osx_utils.sh

```diff
@@ -455,6 +455,8 @@ function macos_arm64_cross_build_setup {
     export BUILD_PREFIX=/opt/arm64-builds
     sudo mkdir -p $BUILD_PREFIX
     sudo chown -R $USER $BUILD_PREFIX
+    mkdir -p $BUILD_PREFIX/lib
+    mkdir -p $BUILD_PREFIX/include
```

Comment:
```suggestion
    sudo mkdir -p $BUILD_PREFIX/lib $BUILD_PREFIX/include
    sudo chown -R $USER $BUILD_PREFIX
```
This might be simpler?